src/main/java/com/hotels/bdp/circustrain/bigquery/extraction/container/UpdatePartitionSchemaAction.java

```diff
@@ -28,22 +29,26 @@
   private final Storage storage;
   private final ExtractionUri extractionUri;
   private final SchemaExtractor schemaExtractor;
+  private final Table sourceTableAsHive;
 
   public UpdatePartitionSchemaAction(
       Partition partition,
       Storage storage,
       ExtractionUri extractionUri,
-      SchemaExtractor schemaExtractor) {
+      SchemaExtractor schemaExtractor,
+      Table sourceTableAsHive) {
```

Comment:
I think this would read better if this was the first argument in the constructor - that way we keep the two hive parameters next to each other. This is a breaking change to the constructor anyway so might as well order them how we like.